app/controllers/backend/tax_declarations_controller.rb

```diff
@@ -64,7 +64,12 @@ def index
 
       respond_to do |format|
         format.html do
-          render "alert_no_VAT_declaration" unless FinancialYear.where.not(tax_declaration_mode: "none").any?
+          not_opened_financial_year = FinancialYear.opened.empty?
+          opened_financial_year_with_tax_declaration_mode_none = !FinancialYear.with_tax_declaration.include?(FinancialYear.current)
```

Comment:
The variable name doesn't match its content. 
Also can we remove the `!` at the beginning and use the opposite predicate instead (`exclude?`)